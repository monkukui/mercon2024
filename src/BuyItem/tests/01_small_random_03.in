4
517339418 894070271 80
7398400 308586319 24
549619963 700868863 45
495546675 269848655 94
