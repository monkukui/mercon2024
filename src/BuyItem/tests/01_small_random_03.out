4
269848655 495546675
308586319 7398400
700868863 549619963
894070271 517339418
